packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -194,6 +194,14 @@ export class OdspDeltaStorageWithCache implements IDocumentDeltaStorageService {
 				validateMessages("cached", messagesFromCache, from, this.logger);
 				if (messagesFromCache.length !== 0) {
```

Comment:
unrelated to your changes, but it feels like `this.firstCacheMiss` should be set to `from` (or maybe even -1 to disable any future reads) if we did not get any ops

Reply:
We are already setting from "from" in case there are no ops. Yes but if set from, -1 either we got the ops or not, it will prevent a read but logically that is not correct as miss happened at "from" and not a from -1.

Reply:
But we can afford to set it to -1 from actual number as no big deal. It is just 1 op at the end of day.

Reply:
Made the changes. Please check @vladsud